app/(modelsMap)/models/modeling.tsx

```diff
@@ -24,15 +24,23 @@ export const ModelingPage = () => {
 
   return (
     <>
-      <Row>
-        <Col md={3} style={{ padding: 0 }}>
+      <Row style={{ paddingLeft: "10px" }}>
+        <Col
+          md={3}
+          height="55vh"
+          style={{
+            padding: "5px",
+            border: "1px solid #0000002d",
+            borderRadius: "10px",
```

Comment:
It may just be me, but for some reason it doesn't look right when catalog listings next to maps are rounded, in the way that is ok for info boxes. [Generally](http://nextgen-dev.ioos.us/xreds/) catalogs have a bit less styling because they are info dense

Reply:
Sounds good! Yeah, don't 100% know the usual design standards, just mimicking what I've seen on the rest of mariners and wanting to create some spacing so its easier to digest when first arriving at the page. I'll remove the roundedness for the catalog!